Cargo.toml

```diff
@@ -22,12 +22,11 @@ categories = [
 exclude = ["ci/*", ".github/*"]
 
 [features]
-default = ["log/release_max_level_off"]
 
 [dependencies]
 cfg-if = "1.0"
 rand = "0.8"
-log = { version = "0.4", optional = true }
+log = "0.4"
```

Comment:
It may be preferrable to leave `log` as optional, but add it to the default feature set. Kind of like

```toml
[features]
default = ["log"]

[dependencies]
# ...
log = { version = "0.4", optional = true }
```

Reply:
Oh, okay, good to know. Thanks!